examples/auto_enabled_basic.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
Can you add a link to this example in the readme? https://github.com/polleverywhere/sordid-dragon/blob/master/README.md#development